src/pages/components/timeline.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Timeline
+description: >
+  The Timeline component lists events over a period of time.
```

Comment:
For consistency with almost all other component pages, "A Timeline lists..." vs ~The Timeline component~